playbooks/ansible-test-network-integration-base/pre.yaml

```diff
@@ -11,6 +11,10 @@
         _network_os: "{{ hostvars[groups['openvswitch'][0]]['ansible_network_os'] }}"
       when: '"openvswitch" in groups'
 
+    - name: Ensure tox
```

Comment:
I feel this is fine, again we should only need to limit to aws nodes. As tox is managed via DIB.  The fear I have, is we overwrite DIBs setup and manage if tox